metadata-service/openapi-servlet/src/main/java/io/datahubproject/openapi/operations/elastic/OperationsController.java

```diff
@@ -42,6 +45,9 @@ public class OperationsController {
   @Autowired
   @Qualifier("elasticSearchSystemMetadataService")
   private SystemMetadataService _systemMetadataService;
+  @Autowired
+  @Qualifier("timeseriesAspectService")
+  private TimeseriesAspectService _timeseriesAspectService;
```

Comment:
Is there an existing test that at least validates that this works?

Reply:
There isn't a Spring loading test set up for this module, @iprentic you should be able copy something similar to what the consumers do with their [Spring context loading tests](https://github.com/datahub-project/datahub/blob/78923e8cfb049972a6295e25e115aefe8479e6b9/metadata-jobs/mae-consumer/src/test/java/com/linkedin/metadata/kafka/hook/spring/MCLSpringTest.java)